i18n/en/design-system.json

```diff
@@ -17,6 +17,8 @@
   "global-footer-advertise-link-media-kit": "Media Kit",
   "global-footer-community-apps-description": "Take your favorite fandoms with you and never miss a beat",
   "global-footer-community-apps-header": "Community Apps",
+  "global-footer-fandom-app-description": "Take your favorite fandoms with you and never miss a beat",
```

Comment:
`this translation will be the same as 
"global-footer-community-apps-description" key above, we've added new keys because of caching reasons